applications/tari_console_wallet/src/main.rs

```diff
@@ -137,6 +149,36 @@ fn main_inner() -> Result<(), ExitCodes> {
     result
 }
 
+fn get_peers(config: &GlobalConfig) -> Result<(Peer, Peer), ExitCodes> {
```

Comment:
I don't think we should have 2 answers here? The base node we monitor should also be the base node we are using to broadcast TX and validate UTXOs

Reply:
Yeah that makes sense, and I did have it that way previously. I must have mistaken previous discussion saying they should be separate. Anyway, I'll update it.